### DISCUSSION TITLE
Questions about customization not working when loading a script I created

### DISCUSSION THREAD

**Discussion by qwikster** [rocksdanister/lively#2359] [Q&A]
Okay, first, I'm new to JavaScript and development in general, bear with me -
Continuing, I'm not sure why, but my new wallpaper does not seem to be able to apply configs. You can view it at https://qwikster.dev/stars or the github at https://github.com/OCO-Inc/homepage .

Any help with this is appreciated - I've spent far too long on figuring this out. I'm probably in way over my head...

I'm loading stars.html inside Lively. The config options appear fine, and can be changed, but they do not reflect on the wallpaper, even after both restarting the app and swapping between wallpapers a few times to restart it. I've also checked my code for spelling mistakes, and it should work fine either way...

I'm not sure how to directly call the config when testing. I can't get in the right scope for it

**Comment by rocksdanister** [comment]
https://github.com/OCO-Inc/homepage/blob/acc40f8a0cdb80700a32af5b839a782e127f648d/script.js#L1

The `livelyPropertyListener` function appears to be inside `window.onload` ?

**Reply by qwikster** [reply]
I honestly have no idea where I'm supposed to be putting things, or what goes inside `window.onload`. Kind of assumed it was similar to the <html> tag, or something like that, that you have as a completely outer container.

So uh, this is a bit embarrassing, and I definitely should have mentioned it in the first post, but I ~~mutillated your code~~ based my code from your four year old post at https://www.reddit.com/r/LivelyWallpaper/comments/gvm9dk/attractive_particles_wallpaper/

I did plan to give credit where it was due - i promise - IIRC the way you originally coded had everything encased in `window.onload`

**Reply by rocksdanister** [reply]
The original project is not mine.
I sent a pr with fix, basically just removed onload.
